src/rawdog/__main__.py

```diff
@@ -35,9 +36,14 @@ def rawdog(prompt: str, config, llm_client):
                 print(message)
         except KeyboardInterrupt:
             error = "Execution interrupted by user"
+        except Exception as e:
+            error = str(e)
```

Comment:
Now that the python script is run in a subprocess there aren't really very many exceptional cases. Basically just litellm errors and dryrun rejected scripts. In my opinion we shouldn't try to recover from either of them. I think if the user doesn't accept the edits instead of throwing an exception we should break from the loop. Presumably if they didn't like the llm's first attempt they might rather give new information. If they do want to rerun exactly they can ask the same question again.